### PR TITLE
Research-page-update

### PR DESCRIPTION
**Pull request by matthewmccarthy11**
I had to make a new pull request- the other one had a load of documents added to it. I'm assuming it was some sort of mistake, unless someone else was using the branch to add or move around a bunch of documents, somewhere around 300? 

But took the comments into account @rufuspollock and now they are on the main page. Will hopefully be done with a blog post on 2023 by the start of meeting tomorrow as well.

### CODE REVIEW

File: content/research.md
Comment (catherinet1) [inline content/research.md:1]: Confused about the frontmatter, doesn't seem accurate to this page... ?

Comment (catherinet1) [reply content/research.md:1]: Ah, or I'm guessing that this perhaps will show up as a project list or something like this on the live page.

File: content/research.md [hunk 1]
```diff
@@ -86,10 +86,25 @@ ourwork:
 
 import { ResearchTopicMap, ProjectsList } from './components/custom/Research';
 
-We are a **research collective** exploring paths to [awakening societies](/awakening-society) 🌻. We produce reports, papers, methods and tools. We [welcome participation](/research-community).
+## About
+
+We are a **research collective** exploring paths to [awakening societies](/awakening-society) 🌻. We produce reports, papers, methods and tools. We welcome participation and [more info on our research community can be found here.](/research-community)
```
Comment (catherinet1) [inline content/research.md:91]: Do the awakening society and research community links go somewhere? When I view the file in the GitHub repo and click on the links they don't take me to another page, but maybe they will on the live version of the page?

**Review by catherinet1 — COMMENTED** [review]
Overall, this looks fine to me for now. I'd let @rufuspollock also take a look, since I don't know what the comments were that Rufus left on the previous pull request. Thanks for this work, Matthew!

**Comment by catherinet1** [comment]
Ah, yes, and just wanting to flag that we are from latest discussions favouring the language of 'ecosystem' over 'movement'.

**Comment by rufuspollock** [comment]
I've now incorporated key parts of this into a change and pushed that.

**Pull request closed without merging**